virtcontainers/container.go

```diff
@@ -515,10 +515,11 @@ func (c *Container) mountSharedDirMounts(hostSharedDir, guestSharedDir string) (
 	var sharedDirMounts []Mount
 	var ignoredMounts []Mount
 	for idx, m := range c.mounts {
-		if isSystemMount(m.Destination) {
-			if !(IsDockerVolume(m.Source) || Isk8sHostEmptyDir(m.Source)) {
-				continue
-			}
+		// Skip mounting certain system paths from the source on the host side
+		// into the container as it does not make sense to do so.
+		// Example sources could be /sys/fs/cgroup etc.
+		if isSystemMount(m.Source) {
```

Comment:
@awprice Your comment makes sense, we do not mount certain system paths from the host side into the guest, however it makes sense to do so for system paths that are docker volumes and k8s empty dirs.

Reply:
We are using containerd, maybe we need to update the logic to handle docker volumes, k8s empty dirs and containerd volumes?

I don't think containerd volumes have been factored in here and that's probably why we are running into this issue, as we don't use docker.